mage_ai/streaming/sources/nats_js.py

```diff
@@ -0,0 +1,188 @@
+import asyncio
```

Comment:
Any reason for adding "_js" in the file name?
Rest of the files in this directory are <streaming_source_technology>.py

Reply:
I was anticipating the possibility of another implementation of this that used the non-jetstream version of NATS.